prepare.xml

```diff
@@ -0,0 +1,116 @@
+<project name="Idea Haxe language scripted package plugin" default="init" basedir=".">
+
+
+    <property name="grammarKitVersion" value="1.2.0"/>
+    <property name="toolsDir" value="${basedir}/tools/"/>
+
+    <property name="ideaVersion" value="${version}"/>
+    <property name="ideaIUFile" value="${toolsDir}/ideaIU-${ideaVersion}.tar.gz"/>
```

Comment:
So, this downloads the linux build.  Should we, instead, download the platform-specific build.  (e.g. .zip for Windows, .dmg for macOS)?

Reply:
i could get the windows version for windows and unzip but i don't know of any ant task that unpacks .dmg files. i do however think it would be a bit of over engineering, as far as i know we only use the java files which are cross-platform anyway. (i use the Linux files when building on windows without any problem)

Reply:
Yes, and no.  For the time being, it's OK, because we can generally use the Java that is running on a system.  The default installation of Java is what we're using for Travis, too.

For the long term, we should be using the version of Java that JetBrains delivers as part of their release.  That version specially modified to fix bugs that JetBrains' products run into, and are platform-specific.